src/libs/ReportUtils.ts

```diff
@@ -5852,18 +5852,17 @@ function getDeletedTransactionMessage(action: ReportAction) {
     return message;
 }
 
-function getReportDetails(reportID: string): {reportName: string; reportUrl: string} {
-    const report = allReports?.[`${ONYXKEYS.COLLECTION.REPORT}${reportID}`];
+function getReportDetails(reportID: string, report?: Report): {reportName: string; reportUrl: string} {
     return {
         reportName: getReportName(report) ?? report?.reportName ?? '',
```

Comment:
`getReportName` is not a pure function, so `getReportDetails` is also still not pure